src/github-actions-reporter.js

```diff
@@ -0,0 +1,36 @@
+class GithubActionsReporter {
```

Comment:
We could hook this up to Sonar for proper reporting, and we wouldn't need to add this class.